packages/plugin-ext/src/plugin/languages/call-hierarchy.ts

```diff
@@ -0,0 +1,104 @@
+/********************************************************************************
+ * Copyright (C) 2019 Red Hat, Inc. and others.
```

Comment:
2020

Reply:
Haha...this is a really old PR originally...but of course you're right.